obspy/taup/tau.py

```diff
@@ -31,6 +58,131 @@ def __str__(self):
     def __repr__(self):
         return "[%s]" % (", ".join([repr(_i) for _i in self]))
 
+    def plot(self, plot_type="spherical", plot_all=True, ax=None, show=True):
+        """
+        Plot the ray paths if any have been calculated.
+
+        :param plot_type: Either ``"spherical"`` or ``"cartesian"``.
+            A spherical plot is always global whereas a cartesian one can
+            also be local.
+        :type plot_type: str
+        :param plot_all: By default only rays that match the requested
```

Comment:
Ah true...i changed the default behavior in some commit and forgot to adopt the doc. thanks.